packages/static-wado-creator/lib/operation/ThumbnailService.js

```diff
@@ -17,9 +19,13 @@ function getThumbIndex(listThumbs) {
 function internalGenerateThumbnail(originalImageFrame, dataset, metadata, transferSyntaxUid, doneCallback) {
   decodeImage(originalImageFrame, dataset, transferSyntaxUid)
     .then((decodeResult = {}) => {
-      const { imageFrame, imageInfo } = decodeResult;
-      const pixelData = dicomCodec.getPixelData(imageFrame, imageInfo, transferSyntaxUid);
-      staticCS.getRenderedBuffer(transferSyntaxUid, pixelData, metadata, doneCallback);
+      if (isVideo(transferSyntaxUid)) {
+        console.log("Video data - no thumbnail generator yet");
```

Comment:
Just drop a message for now - eventually it would be good to call out to ffmpeg to generate the data.